packages/components/src/combobox-control/index.tsx

```diff
@@ -241,9 +241,6 @@ function ComboboxControl( props: ComboboxControlProps ) {
 		if ( expandOnFocus ) {
 			setIsExpanded( true );
 		}
-
-		onFilterValueChange( '' );
-		setInputValue( '' );
```

Comment:
@adamsilverstein @youknowriad, do we have any context why that was added? I'm struggling to understand it.

Reply:
I think I understand it now. We're clearing the input value so all the suggestions show up. But that's a bit weird because there is already a selected value, so why not narrow the options to this value and the user can progressively see more as they delete characters from the input?